src/SFA.DAS.RoATPService.Application.Api/Controllers/LookupDataController.cs

```diff
@@ -32,14 +32,31 @@ public async Task<IActionResult> ProviderTypes()
         }
 
         [HttpGet]
-        [Route("organisationTypes")]
+        [Route("organisationTypes/{providerTypeId}")]
```

Comment:
Is this a breaking change for the API clients in admin service?

Reply:
https://github.com/SkillsFundingAgency/das-admin-service/blob/a1748d828e746d782d99a7ba476441c3bca0b5ab/src/SFA.DAS.AdminService.Web/Infrastructure/RoatpApiClient.cs#L56